packages/effect/src/Effect.ts

```diff
@@ -5958,12 +5958,13 @@ export declare namespace Tag {
    * @category models
    */
   export interface ProhibitedType {
-    _op?: "propety name _op is forbidden"
-    _tag?: "propety name _tag is forbidden"
-    of?: "propety name of is forbidden"
-    context?: "propety name context is forbidden"
-    key?: "propety name key is forbidden"
-    stack?: "propety name stack is forbidden"
+    _op?: `propety "_op" is forbidden`
```

Comment:
`property` typo?

Reply:
Nah man, that's definitely how it's spelled!

Reply:
lol

Reply:
https://github.com/Effect-TS/effect/pull/3345